src/evergreen/project_history.py

```diff
@@ -0,0 +1,40 @@
+# -*- encoding: utf-8 -*-
+"""Project history representation of evergreen."""
+from __future__ import absolute_import
+
+from evergreen.base import _BaseEvergreenObject, evg_attrib
+
+
+class Version(_BaseEvergreenObject):
```

Comment:
There is already a `Version` object in version.py. Could we rename this to something else?

Reply:
I've renamed it, although the APIs I'm calling appear to be sending back something roughly equivalent to the existing `Version`, just with some discrepancies in field name and less information.